src/index.ts

```diff
@@ -1281,127 +1294,130 @@ export function useIsEmulator(): AsyncHookResult<boolean> {
   return useOnMount(isEmulator, false);
 }
 
-export default {
-  getUniqueId,
-  getInstanceId,
-  getInstanceIdSync,
-  getSerialNumber,
-  getSerialNumberSync,
+const deviceInfoModule: DeviceInfoModule = {
```

Comment:
DeviceInfoModule was pulled in from an internal type. Now it's being exported, is that exposing an internal-labeled item publicly?

Reply:
`DeviceInfoModule` is used in the `index.ts` file, but not exposed to the user (totally absent in the generated `index.d.ts`)